.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
 <!--
-Thanks for sending a pull request!
+Thank you for contributing! Please remember to:
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
+- Read our contributing guidelines: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
+- Sign your commits with your real name. You can amend previous commits with `git commit -s --amend`.
+-->
+
+#### Description
```

Comment:
I don't see the point of the header, the way github works it copies the commit description by default as first thing in the field so now you would require me to copy paste my entire commit description each time to below that header. Overall I don't see how this title improves anything so I rather just not have it at all.